game/plugins/src/main/kotlin/gg/rsmod/plugins/content/inter/currenttime/current_time.plugin.kts

```diff
@@ -0,0 +1,19 @@
+package gg.rsmod.plugins.content.inter.currenttime
+
+import gg.rsmod.game.model.interf.DisplayMode
+import java.util.Calendar
+
+DisplayMode.values.forEach { mode ->
+    val child = when (mode) {
```

Comment:
Since this is now defined in multiple places, we want to consider defining it somewhere and accessing that instead. Otherwise if this ever changes, we will need to change it in different places.